dask_expr/datasets.py

```diff
@@ -31,19 +38,29 @@ class Timeseries(PartitionsFiltered, BlockwiseIO):
         "seed": None,
         "kwargs": {},
         "_partitions": None,
+        "_series": False,
+        "columns": None,
     }
 
     @functools.cached_property
     def _meta(self):
-        dtypes = self.operand("dtypes")
+        dtypes = self._dtypes
         states = [0] * len(dtypes)
-        return make_timeseries_part(
+        result = make_timeseries_part(
             "2000", "2000", dtypes, list(dtypes.keys()), "1H", states, self.kwargs
         ).iloc[:0]
+        if self._series:
+            return result[result.columns[0]]
+        return result
 
     def _divisions(self):
         return pd.date_range(start=self.start, end=self.end, freq=self.partition_freq)
 
+    @property
+    def _dtypes(self):
+        dtypes = self.operand("dtypes")
+        return {col: dtypes[col] for col in self.operand("columns")}
```

Comment:
What if `self.operand("columns")` is `None`?

Reply:
Good point, should not happen. Adjusted the signature